src/agent/src/ap.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) IBM Corp. 2022
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::fmt;
+use std::str::FromStr;
+
+use anyhow::{anyhow, Context};
+
+// IBM Adjunct Processor (AP) is the bus used by IBM Crypto Express hardware security modules on
+// IBM Z & LinuxONE (s390x)
```

Comment:
fyi, I harvested this for the [CC glossary](https://github.com/confidential-containers/documentation/wiki/Glossary#adjunct-processor) :smile: